test/lib/i18n_test.rb

```diff
@@ -135,6 +142,16 @@ def check_values_for_raw_html(hash)
     end
   end
 
+  def check_values_for_gender_macro(hash)
+    hash.each_pair do |k, v|
+      if v.is_a? Hash
+        check_values_for_gender_macro(v)
+      else
+        raise "Avoid using GENDER macro in '#{k}: #{v}'" if v.include? "{{GENDER:"
```

Comment:
This would be a good way to detect the broken `{{PLURAL` syntax I spotted too, since the correct syntax on Translatewiki.net populates different entries in our YAML file instead of leaving that token around.

Reply:
I've updated it to handle `PLURAL` and `GRAMMAR` as well now.